app/lib/katello/resources/candlepin.rb

```diff
@@ -76,6 +76,8 @@ class UpstreamCandlepinResource < CandlepinResource
         self.prefix = '/subscription'
 
         class << self
+          delegate :[], to: :json_resource
```

Comment:
Yes, good. And [here](https://github.com/jturel/katello/blob/df26c17f0c793c0e8a4e7ad84aa02e05099bc823/app/lib/katello/resources/candlepin/upstream_pool.rb#L12)?